proto/axelar/evm/v1beta1/query.proto

```diff
@@ -156,3 +156,36 @@ message BytecodeRequest {
 }
 
 message BytecodeResponse { string bytecode = 1; }
+
+enum TokenType {
+  option (gogoproto.goproto_enum_prefix) = false;
+  option (gogoproto.goproto_enum_stringer) = true;
+
+  TOKEN_TYPE_UNSPECIFIED = 0
+      [ (gogoproto.enumvalue_customname) = "Unspecified" ];
+  TOKEN_TYPE_INTERNAL = 1 [ (gogoproto.enumvalue_customname) = "Internal" ];
+  TOKEN_TYPE_EXTERNAL = 2 [ (gogoproto.enumvalue_customname) = "External" ];
+}
+
+message ERC20TokensRequest {
+  string chain = 1;
+  TokenType type = 2;
+}
+
+message ERC20TokensResponse { repeated string assets = 1; }
+
+message TokenInfoRequest {
+  string chain = 1;
+  oneof find_by {
+    string asset = 2;
+    string symbol = 3;
+  }
+}
+
+message TokenInfoResponse {
+  string asset = 1;
+  TokenDetails details = 2 [ (gogoproto.nullable) = false ];
+  string address = 3;
+  bool confirmed = 4;
+  bool is_external = 5;
+}
```

Comment:
i guess this renders the `token-address` query obsolete. Should we remove that query in this PR?

Reply:
Since applications still rely on it, we'll deprecate it instead.